service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -49,6 +61,15 @@
 
     private static final String INVALID_MIGRATION_ID = "invalid id";
 
+    @MockBean
+    private TaskListService taskListService;
```

Comment:
I don't know much about these elements, but it seems they only work with the case data itself (not external components).
Shouldn't we use the case data to assert that they this is working (as opposed to using a mock)?
Or is the case that these components are so well tested that just by knowing they're being called, we're already happy with this controller test?

Reply:
yea, they are already tested, so no point to repeat it here. 
Code used in migration is mostly copy of 
https://github.com/hmcts/fpl-ccd-configuration/blob/master/service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseEventHandler.java

which is verified by unit and integration tests, e.g. here 

https://github.com/hmcts/fpl-ccd-configuration/blob/master/service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/TaskListControllerSubmittedTest.java